cloudinit/distros/rhel.py

```diff
@@ -29,6 +29,10 @@ class Distro(distros.Distro):
     network_script_tpl = "/etc/sysconfig/network-scripts/ifcfg-%s"
     tz_local_fn = "/etc/localtime"
     usr_lib_exec = "/usr/libexec"
+    # RHEL and derivatives have the following dhclient lease path:
+    # /var/lib/NetworkManager/dhclient-<uuid>-<network_interface>.lease
+    dhclient_lease_directory = '/var/lib/NetworkManager'
+    dhclient_lease_file_regex = r'dhclient-[\w-]+\.lease'
```

Comment:
where is the equivalent for this in other distros? particularly freebsd

Reply:
Hi @igalic,
As I mentioned earlier, this commit is intended as a WIP, a draft to show an idea how I intend to implement it. I just picked RHEL distro as an example. 

Once I get Brett's approval on this course, I'll implement it for every distro, FreeBSD included.